.gitignore

```diff
@@ -50,7 +50,8 @@ build/.cmake/
 !/mods/minetest/
 /mods/minetest/*
 !/mods/minetest/mods_here.txt
```

Comment:
Btw. whouldn't this be 
```suggestion
!/mods/mods_here.txt
```
?

Reply:
all references to mods/minetest/ are wrong, the folder used to be called that 10 years ago

Reply:
I guess I'll update it, when I'm already here.